src/lib/styled/styleFunctions.ts

```diff
@@ -49,8 +55,15 @@ transition: 200ms color;
   color: ${theme.disabledUiTextColor};
 }`
 
-export const sideBarTextColor = ({ theme }: StyledProps) =>
-  `color: ${theme.sideBarTextColor};`
+export const sideBarTextColor = ({
+  theme
+}: StyledProps) => `color: ${theme.sideBarTextColor};
```

Comment:
The purpose of the style functions is to make easily re-usable content, so that we may be faced to re-use in another component this specific color.
Tying the sidebar Text color to the hover, focus, and active states/classes here defeats the purpose.

Keep the
```
export const sideBarTextColor = ({ theme }: StyledProps) =>
  `color: ${theme.sideBarTextColor};`
``` 
here and 

```&:hover,
  &:focus,
  &:active,
  &.active {
    ${sideBarTextColor}
  }```
in the sideNavigatorItem